core/src/test/java/ai/timefold/solver/core/impl/score/stream/common/uni/AbstractUniConstraintStreamTest.java

```diff
@@ -2414,34 +2415,49 @@ public void expandToQuad() {
     @Override
     @TestTemplate
     public void flattenLastWithDuplicates() {
```

Comment:
Why was this change to a test necessary?
I'd like to understand the extent to which we're actually potentially breaking people.